kentikapi/client.go

```diff
@@ -52,12 +54,11 @@ type Client struct {
 	Query              *resources.QueryAPI
 	SavedFilters       *resources.SavedFiltersAPI
 	Sites              *resources.SitesAPI
-	Synthetics         *resources.SyntheticsAPI
+	Synthetics         *synthetics.API
 	Tags               *resources.TagsAPI
 	Users              *resources.UsersAPI
 
-	// SyntheticsAdmin and SyntheticsData are gRPC clients
-	// for Kentik API Cloud Export and Synthetics services.
+	// SyntheticsAdmin and SyntheticsData are legacy gRPC clients for Kentik API Synthetics services.
```

Comment:
It might not be a good word, but it is better to use new client.Synthetics API than those generated clients directly. Those clients (and comment) will be removed soon.